Sources/RunnerLib/Runtime.swift

```diff
@@ -31,7 +31,7 @@ public struct Runtime {
 
         if marathonScripts != nil {
             // TODO: Support running from a fork?
```

Comment:
<!--
  0 failure: 
  1 warning:  TODOs should be r...
  
  
  DangerID: danger-id-default;
  File: Sources/RunnerLib/Runtime.swift;
  Line: 33;
-->

- :warning: TODOs should be resolved (Support running from a fork?). (`todo`)